yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1055,24 +1056,33 @@ func (yb *TargetYugabyteDB) alterColumns(tableColumnsMap map[string][]string, al
 			query := fmt.Sprintf(`ALTER TABLE %s ALTER COLUMN %s %s`, qualifiedTableName, column, alterAction)
 			batch.Queue(query)
 		}
-
-		err := yb.connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
-			br := conn.SendBatch(context.Background(), &batch)
-			for i := 0; i < batch.Len(); i++ {
-				_, err := br.Exec()
-				if err != nil {
-					log.Errorf("executing query to alter columns for table(%s): %v", qualifiedTableName, err)
-					return false, fmt.Errorf("executing query to alter columns for table(%s): %w", qualifiedTableName, err)
+		sleepIntervalSec := 10
+		for i := 0; i < ALTER_QUERY_RETRY_COUNT; i++ {
+			err := yb.connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
+				br := conn.SendBatch(context.Background(), &batch)
+				for i := 0; i < batch.Len(); i++ {
+					_, err := br.Exec()
+					if err != nil {
+						log.Errorf("executing query to alter columns for table(%s): %v", qualifiedTableName, err)
+						return false, fmt.Errorf("executing query to alter columns for table(%s): %w", qualifiedTableName, err)
+					}
 				}
+				if err := br.Close(); err != nil {
+					log.Errorf("closing batch of queries to alter columns for table(%s): %v", qualifiedTableName, err)
+					return false, fmt.Errorf("closing batch of queries to alter columns for table(%s): %w", qualifiedTableName, err)
+				}
+				return false, nil
+			})
+			if err != nil {
+				log.Errorf("error in altering columns for table(%s): %v", qualifiedTableName, err)
+				if !strings.Contains(err.Error(), "while reaching out to the tablet servers") {
+					return err
+				}
+				log.Infof("retrying after %d seconds for table(%s)", sleepIntervalSec, qualifiedTableName)
+				time.Sleep(time.Duration(sleepIntervalSec) * time.Second)
```

Comment:
it will be fixed constant `10 sec` sleep after every retry, right? Where is backoff